mypy/constraints.py

```diff
@@ -1018,21 +1018,25 @@ def visit_callable_type(self, template: CallableType) -> list[Constraint]:
             param_spec = template.param_spec()
 
             template_ret_type, cactual_ret_type = template.ret_type, cactual.ret_type
+            bool_type = UnionType(
+                [LiteralType(True, cactual_ret_type), LiteralType(False, cactual_ret_type)]  # type: ignore[arg-type]
+            )
```

Comment:
It seems ugly, but I can't find another way to create `Instance` of `builtins.bool`.